### DISCUSSION TITLE
Welcome to continual-learning-ogdplus Discussions!

### DISCUSSION THREAD

**Discussion by MehdiAbbanaBennani** [MehdiAbbanaBennani/continual-learning-ogdplus#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rkiacnhg** [comment]
Hello,
I am trying to run your code, but I get an error in main.py regarding "from tap import Tap".
I dont see tap in the requirements.txt or the folder, and the error tells me  "cannot import name "Tap" from "tap". Would you be able to help me with this error? Also, what would you suggest as an alternative to "SLURM_TMPDIR" for wandb_dir? Thank you.

**Reply by MehdiAbbanaBennani** [reply]
Hello !

Sure, tap corresponds to  https://pypi.org/project/typed-argument-parser/
One alternative to SLURM_TMPDIR is "/tmp"

Hope it helps :)

**Reply by rkiacnhg** [reply]
thank you! I will try this
